runner/process.py

```diff
@@ -75,6 +77,7 @@ def __init__(self,
         sys.stdout.write(" input skipped...")
         sys.stdout.flush()
         self.input_skipped_ci = confidence_slice(self.bootstrap_input_skipped(), "0.99")
+        print
```

Comment:
Forgotten print?

Reply:
It was accidentally removed at some point: without it, it's really difficult to work out when the processing script has moved to processing the next input file. Given that this takes many minutes (for the full dataset, hours) per recoverer, it's nice to know how it's going :)

Reply:
Sounds fair enough.